sources/packages/web/src/components/aest/students/assessment/Request.vue

```diff
@@ -27,8 +27,22 @@
               ><template #body="slotProps">{{
                 dateOnlyLongString(slotProps.data.submittedDate)
               }}</template></Column
-            ><Column field="triggerType" header="Type" sortable="true"></Column
-            ><Column header="Request form"></Column
+            ><Column field="triggerType" header="Type" sortable="true"></Column>
+            <Column header="Request form" sortable="false"
+              ><template #body="slotProps"
+                ><v-btn
+                  @click="
+                    viewRequest(slotProps.data.triggerType, slotProps.data.id)
+                  "
+                  color="primary"
+                  variant="text"
+                  class="text-decoration-underline"
+                >
+                  <font-awesome-icon :icon="['fas', 'file-alt']" class="mr-2" />
```

Comment:
mm, I think `file-alt` regular icons are used in figma, i.e `:icon="['far', 'file-alt']"`. maybe you can double check

Reply:
@ann-aot Yes regular can be used here as it is not a pro icon.

Reply:
Changed
![image](https://user-images.githubusercontent.com/61259237/162072343-6f16e437-98ef-4834-b69f-155051b75b05.png)